internal/connectors/plugins/public/wise/payouts.go

```diff
@@ -45,6 +45,10 @@ func (p *Plugin) createPayout(ctx context.Context, pi models.PSPPaymentInitiatio
 		return models.PSPPayment{}, err
 	}
 
+	if payment == nil {
+		return models.PSPPayment{}, fmt.Errorf("unsupported payments: %w", models.ErrInvalidRequest)
+	}
```

Comment:
Ah sorry if I wasn't clear, but I think this is less optimal than returning an error directly in the `fromPayoutToPayment` and `fromTransferToPayment` functions 😥 

The difference is that if we see this error we go "why is it nil?" and have no clues. Whereas if we get a "unsupported currency error" where we were returning `nil, nil` we have a more maintainable codebase.

A nil check would be great in the case that we want to skip any following actions, but I think in this case there is no circumstance where we'd just return an empty payment, so better to intercept the error as early as possible.

Reply:
Should be good now !